ios/browser/api/ledger/legacy_database/core_data_models/PublisherInfo.h

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
in theory, files with the old-style license header (the proper one has `Copyright (c) 2019 The Brave Authors. All rights reserved.`) shouldn't pass linter

Reply:
Yeah the CoreData models I just ignored linting because these files are deprecated and won't be touched again in the future